actionpack/test/dispatch/content_security_policy_test.rb

```diff
@@ -377,6 +377,24 @@ class PolicyController < ActionController::Base
 
     content_security_policy_report_only only: :report_only
 
+    content_security_policy only: :api do |format|
+      format.json do |p|
+        p.default_src :none
+        p.frame_ancestors :none
```

Comment:
I suggest including [`sandbox`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/sandbox) in the default policy for API requests as well. It's a few more bytes and provides a little more protection against hypothetical situations.